src/App.vue

```diff
@@ -1,12 +1,41 @@
 <template>
   <div id="app">
-    <h1>hello world</h1>
+    <tx-preloader v-if="preload" :progress="progress" />
+
+    <main v-else>
+      <h1>hello world</h1>
+    </main>
   </div>
 </template>
 
 <script>
-export default {}
+import Preloader from "./assets/components/atoms/Preloader.vue";
+
+export default {
+  components: {
+    "tx-preloader": Preloader,
```

Comment:
what is `tx` ?

Reply:
Ok then rename your Placeholder.vue component become `TxPlaceholder`

Reply:
don't differentiate the component naming like this

```
components: {
  "txt-preloader": Preloader
}
```

Do this instead

```
components: {
  TxPreloader
}
```

Reply:
got it ✔️